inc/sainexthopgroup.h

```diff
@@ -59,22 +59,40 @@ typedef enum _sai_next_hop_group_attr_t
     /** Number of next hops in the group [uint32_t] */
     SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_COUNT,
 
+    /** Next hop list [sai_object_list_t] */
+    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST,
+
     /** READ-WRITE */
 
     /** Next hop group type [sai_next_hop_group_type_t] (MANDATORY_ON_CREATE|CREATE_ONLY) */
     SAI_NEXT_HOP_GROUP_ATTR_TYPE,
 
-    /** Next hop list [sai_object_list_t] (MAXDATORY_ON_CREATE) 
-     * The next hop group must have at least one next hop member at the creation time */
-    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_LIST,
```

Comment:
the tushar's suggestion looks ok to me. As about the remove_nexthopgroup api, I assume we need to remove all nexthop members first before removing the nexthop group. @itaibaz , @kcudnik , are you ok with it?